pom.xml

```diff
@@ -79,86 +85,44 @@
                         <artifactId>spring-boot-maven-plugin</artifactId>
                         <executions>
                             <execution>
-                                <!-- Spring Boot Actuator displays build-related information if a META-INF/build-info.properties file is present -->
```

Comment:
Not right anymore?

Reply:
Fixed in c4c688f0202237231ff7b07e7ccb71487bc77458

Reply:
Thanks